src/core/Rule.ml

```diff
@@ -318,6 +318,17 @@ and extract_reduction = Separate | Concat [@@deriving show]
 type header = { name : string; value : string } [@@deriving show]
 type meth = [ `DELETE | `GET | `POST | `HEAD | `PUT ] [@@deriving show]
 
+(* Used to request additional auth headers are computed and added automatically,
```

Comment:
I don't understand this comment.
Maybe an example would help.

Reply:
I'll add something. The gist is that you might want an `Authorization` header (or other additions/modification) with an HMAC or something over the whole HTTP frame, but that might depend on interpolated metavariable values, current UTC time, etc..; so you can't add it yourself.